### PR TITLE
Implement Submission filter

### LINKED ISSUE #1442: Search/filter student submissions
Tutors need an easy way to see all their students' submissions.
Or maybe can have a tab that shows all submissions for the tutor's students?

### PR DESCRIPTION
**Pull request by allenwq**
Fix #1442 

screen shot 2016-09-07 at 6 38 43 pm

Pending to add specs.

### CODE REVIEW

**Comment by fonglh** [comment]
I was under the impression they wanted a JS filter.

File: app/models/course/assessment.rb [hunk 1]
```diff
@@ -38,6 +38,8 @@ class Course::Assessment < ActiveRecord::Base
   has_many :assessment_conditions, class_name: Course::Condition::Assessment.name,
                                    inverse_of: :assessment, dependent: :destroy
 
+  scope :published, -> { where(draft: false) }
```
Comment (fonglh) [inline app/models/course/assessment.rb]: This is for convenience?

**Comment by allenwq** [comment]
They want the same thing as V1.  And the JS won't work because it's not a UI thing, need to query the DB for the results which might not be displayed in the current page.

File: app/models/course/assessment.rb (hunk 1 above)
Comment (allenwq) [inline app/models/course/assessment.rb]: Think it would be useful to have a published scope. rails has provided a `draft` scope, but cannot be used here.

**Comment by allenwq** [comment]
Specs added in #1462

**Pull request merged**